apps/scale-cluster/scale-cluster-cf.yml.j2

```
@@ -126,3 +141,50 @@ Resources:
       Action: lambda:InvokeFunction
       Principal: events.amazonaws.com
       SourceArn: !GetAtt Schedule.Arn
+
+  {% for job_type, job_spec in job_types.items() if 'Default' != job_spec['compute_environment']['name'] %}
+  {% set name = job_spec['compute_environment']['name'] %}
+  {{ name }}LogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub {{ "/aws/lambda/${"+ name +"Lambda}" }}
```

Extra spaces makes it more readable for me.
```suggestion
      LogGroupName: !Sub {{ "/aws/lambda/${" + name + "Lambda}" }}
```